ctx.go

```diff
@@ -1752,7 +1754,7 @@ func (c *Ctx) SendStream(stream io.Reader, size ...int) error {
 
 // Set sets the response's HTTP header field to the specified key, value.
 func (c *Ctx) Set(key, val string) {
-	c.fasthttp.Response.Header.Set(key, val)
+	c.fasthttp.Response.Header.Add(key, val)
```

Comment:
would not miss this, why is it necessary ?

Reply:
> would not miss this, why is it necessary ?

Yeah i changed it by mistake 🤦